src/app/base/components/export/export.component.ts

```diff
@@ -353,7 +354,8 @@ export class ExportComponent implements OnInit, OnChanges {
     const jsonString = this.exportHelper.buildExportData();
     if (this.exportHelper.error.length != 0) return
     this.downloadState = DownloadState.PREPARATION;
-    this.projectApolloService.prepareRecordExport(this.projectId, jsonString).pipe(first()).subscribe((x) => {
+    if (this.key == '') this.key = null;
```

Comment:
with this we change the class instance of key. What I meant was 

```suggestion
    let keyToSend = this.key;
    if (!keyToSend) = keyToSend = null;
```
or some other variation of this behaviour to ensure that the class instance isn't changed but the values is "skipped" when send to backend